datumaro/plugins/data_formats/mapillary_vistas/base.py

```diff
@@ -34,8 +33,21 @@
 
 class _MapillaryVistasBase(SubsetBase):
     def __init__(
-        self, path, task, subset=None, use_original_config=False, keep_original_category_ids=False
+        self,
+        path,
+        task,
+        subset=None,
+        use_original_config=False,
+        keep_original_category_ids=False,
+        format_version="v2.0",
+        parse_polygon=False,
     ):
+        if format_version == "v1.2" and parse_polygon is True:
+            raise ImportError(
+                "Format version %s is not available for polygons. "
+                "Please try with v2.0 for parsing polygons." % format_version
+            )
```

Comment:
How about to use f-string?
```suggestion
            raise ImportError(
                f"Format version {format_version} is not available for polygons. "
                "Please try with v2.0 for parsing polygons."
            )
```

Reply:
I updated.